src/resources/user_address/spec/user_address.spec.js

```diff
@@ -21,6 +21,15 @@ const address = {
   country: 'My country',
   zipcode: '123456',
 };
+const addressUpdate = {
```

Comment:
Please rename this to `newAddress` or `updatedAddress` or anything more descriptive